bsp/esp-box-3/idf_component.yml

```diff
@@ -1,7 +1,7 @@
 
-version: "1.1.1"
+version: "1.1.1~1"
```

Comment:
To all boards: It is not necessary to bump the version now.

The examples in already uploade versions work fine. And this restructuralization won't affect them

Reply:
The link to component was changed and when you click to link in Component Registry, the link will be bad.

Reply:
hmm, that is right